server/rest.py

```diff
@@ -241,18 +241,25 @@ def deleteTiles(self, item, params):
                 }
 
             # If this file was created by the worker job, delete it
-            if 'largeImageJobId':
+            if 'largeImageJobId' in item:
```

Comment:
Thanks for fixing this whole block. I threw this in at the last moment, and I don't know what I was thinking...this didn't even make sense before.